mmv1/products/redis/Cluster.yaml

```diff
@@ -0,0 +1,194 @@
+  - !ruby/object:Api::Type::Array
+    name: 'discoveryEndpoints'
+    description: |
+      Output only. Endpoints created on each given network,
+      for Redis clients to connect to the cluster.
+      Currently only one endpoint is supported.
+    output: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'address'
+          description: |
+            Output only. Network address of the exposed Redis endpoint used by clients to connect to the service.
+        - !ruby/object:Api::Type::Integer
+          name: 'port'
+          description: |
+            Output only. The port number of the exposed Redis endpoint.
+        - !ruby/object:Api::Type::NestedObject
+          name: 'pscConfig'
+          description: |
+            Output only. Customer configuration for where the endpoint
+            is created and accessed from.
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'network'
+              description: |
+                The consumer network where the network address of the discovery
+                endpoint will be reserved, in the form of
+                projects/{network_project_id}/global/networks/{network_id}.
+  - !ruby/object:Api::Type::Array
+    name: 'pscConnections'
+    description: |
+      Output only. PSC connections for discovery of the cluster topology and accessing the cluster.
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'pscConnectionId'
+          description: |
+            Output only. The PSC connection id of the forwarding rule connected to the service attachment.
+        - !ruby/object:Api::Type::String
+          name: 'address'
+          description: |
+            Output only. The IP allocated on the consumer network for the PSC forwarding rule.
+        - !ruby/object:Api::Type::String
+          name: 'forwardingRule'
+          description: |
+            Output only. The URI of the consumer side forwarding rule. Example: projects/{projectNumOrId}/regions/us-east1/forwardingRules/{resourceId}.
+        - !ruby/object:Api::Type::String
+          name: 'projectId'
+          description: |
+            Output only. The consumer projectId where the forwarding rule is created from.
+        - !ruby/object:Api::Type::String
+          name: 'network'
+          description: |
+            The consumer network where the IP address resides, in the form of projects/{projectId}/global/networks/{network_id}.
+    output: true
+  - !ruby/object:Api::Type::Integer
+    name: replicaCount
+    description: |
+      Optional. The number of replica nodes per shard.
+    required: false
+  - !ruby/object:Api::Type::Integer
+    name: sizeGb
+    description: |
+      Output only. Redis memory size in GB for the entire cluster.
+    output: true
+  - !ruby/object:Api::Type::Integer
```

Comment:
Want to add one more property but wondering how to https://yaqs.corp.google.com/eng/q/4303515998905958400. Could you help?

Reply:
Thanks for your response on yaqs. updated accordingly.